ansible_collections/arista/avd/requirements-dev.txt

```diff
@@ -1,8 +1,3 @@
-# Installing PyAVD as editable install.
-# Providing config-settings in a requirements file is only supported by very recent pip versions.
-# The editable_mode=compat is required for pylance to work correctly.
-# The path is relative to the root of the repo, so this only works when executed from there.
--e python-avd --config-settings editable_mode=compat
 ansible-core>=2.15.0,<2.18.0
```

Comment:
I think we need to add pyavd minimum requirements here, since pyavd may not be installed at all.

Reply:
added